CONTRIBUTING.md

```diff
@@ -49,20 +61,26 @@ The output will be:
 core.hookspath=./.hooks
 ```
 
-## Running rustfmt
+**Note**: The pre-commit hook will not prevent commits if style issues are present, instead it will indicate any files that need formatting, and it will automatically run `rustfmt` on your changed files, each time when you try to do `git commit`.
+
+## Running rustfmt manually
 
 You can run rustfmt (i.e. rustfmt-preview) on one file or on all files.
 
-First try a dry-run on a file you've worked on, say:
-`rustfmt --write-mode diff -- client.rs`
+For example:
+```
+rustfmt client.rs
+```
+
+**Notes**:
+1. *Please add the rustfmt corrections as a separate commit at the end of your Pull Request to make the reviewers happy.*
 
-Any errors or rustfmt failures? Fix those manually.
+2. *most important: `rustfmt` is just a tool to make your code having pretty formatting, your changed code itself importance is definitely above the code format, since anyway the formatting can be done by running `rustfmt` later. So the point is, if not sure what should do on this, or the `rustfmt` is changing a lot of untouched code, please feel free to ignore it. We like to make you also happy to contribute on this open source project!*
```

Comment:
I understand the intention but the wording is confusing.

Reply:
ha ha~   I'm not a good english writer, so perhaps somebody help to rewrite this paragraph?  just want to ease the reader about the minor format issue.